### PR TITLE
Enable RBAC e2e test for pilot v2.

### PR DESCRIPTION
**Pull request by diemtvu**
Issue: https://github.com/istio/istio/issues/5331

### CODE REVIEW

**Comment by diemtvu** [comment]
The root cause is deeper. Put this on hold for now until CDS config fix from @ostromart

File: install/kubernetes/helm/istio/values-envoyv2-transition.yaml [hunk 1]
```diff
@@ -3,7 +3,8 @@ global:
   nodePort: true
   proxy:
     image: proxyv2
-
```
Comment (costinm) [inline install/kubernetes/helm/istio/values-envoyv2-transition.yaml]: This is used in the NO AUTH test - only set it in the auth test ( where both mtls and control plane tls are enabled)

**Comment by ostromart** [comment]
Looks like costin piggybacked the changes we discussed in https://github.com/istio/istio/pull/5127, which points the mixer filter to the correct port based on mesh setting.

**Comment by costinm** [comment]
I think this was fixed by your other PR. Closing.

**Comment by diemtvu** [comment]
No it's not. The other add authn policy test in non-authn mode. This PR (re)enable RBAC test for v2.

**Comment by diemtvu** [comment]
Taken care by https://github.com/istio/istio/pull/5544

**Pull request closed without merging**